.storybook/assets/Appshell.tsx

```diff
@@ -2,7 +2,7 @@ import React, { ReactNode } from "react";
 
 export function AppShell({ children }: { children: ReactNode }) {
 	return (
-		<div style={{ minHeight: "100vh", display: "flex", flexDirection: "column", width: "100%" }}>
+		<div style={{ minHeight: "1000px", display: "flex", flexDirection: "column", width: "100%" }}>
```

Comment:
Just out of curiosity, this 1000px instead of a full vh is ok? 🤔

Reply:
Its just for the storybook preview, if I put it 100vh I need to scroll xD I just dont want to scroll when viewing it, so i set it to a fixed height 😅